doc/nrf/releases_and_maturity/releases/release-notes-2.9.0.rst

```diff
@@ -0,0 +1,870 @@
+See :ref:`nrf9160_gs_updating_fw_modem` for instructions.
+
+Modem-related libraries and versions
+====================================
+
+.. list-table:: Modem-related libraries and versions
+   :widths: 15 10
+   :header-rows: 1
+
+   * - Library name
+     - Version information
+   * - Modem library
+     - `Changelog <Modem library changelog for v2.9.0_>`_
+   * - LwM2M carrier library
+     - `Changelog <LwM2M carrier library changelog for v2.9.0_>`_
+
+
+Known issues
+************
+
+Known issues are only tracked for the latest official release.
+See `known issues for nRF Connect SDK v2.9.0`_ for the list of issues valid for the latest release.
+
+Migration notes
+***************
+
+See the `Migration guide for nRF Connect SDK v2.9.0`_ for the changes required or recommended when migrating your application from |NCS| v2.8.0 to |NCS| v2.9.0.
+
+.. _ncs_release_notes_290_changelog:
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+IDE, OS, and tool support
+=========================
+
+* Updated the deprecation notes for `nRF Command Line Tools`_ added in the previous release.
+  The notes now clearly state the tools will be archived: no updates will be made to the software, but it will still be available for download.
+
+Board support
+=============
+
+* Updated various tests and samples to use Zephyr's :ref:`native_sim<zephyr:native_sim>` instead of Zephyr's :ref:`native_posix<zephyr:native_posix>` for :ref:`running_unit_tests`.
+  This mirrors the deprecation of ``native_posix`` in Zephyr.
+  Support for ``native_posix`` will be removed in Zephyr with the v4.2 release.
+  In the |NCS|, it will be removed once Zephyr v4.2 is upmerged to sdk-nrf.
+
+Build and configuration system
+==============================
+
+* Fixed the issue in the ``nordic-bt-rpc`` snippet where an invalid memory map was created for nRF54H20 devices, which resulted in a runtime failure.
+
+Bootloaders and DFU
+===================
+
+* Updated the allowed offset for :ref:`doc_fw_info` in the :ref:`bootloader`.
+  It can now be placed at offset ``0x600``.
+  However, you cannot use it for any applications with |NSIB| compiled before this change.
+
+Developing with nRF91 Series
+============================
+
+* Moved the Thingy:91 and Thingy:91 X guides to new sections, :ref:`thingy91_ug_intro` and :ref:`ug_thingy91x` respectively, under :ref:`ug_app_dev`.
+
+Developing with nRF70 Series
+============================
+
+|no_changes_yet_note|
+
+Working with nRF54H Series
+==========================
+
+|no_changes_yet_note|
+
+Developing with nRF54L Series
+=============================
+
+* Added the :ref:`nRF54l_signing_app_with_flpr_payload` page that includes instructions for building separate applications, merging them, and signing them for MCUboot.
+
+Developing with nRF53 Series
+============================
+
+|no_changes_yet_note|
```

Comment:
Please add entry for https://github.com/nrfconnect/sdk-nrf/pull/19390:

```suggestion
* Moved the Thingy:53 to a new section, :ref:`ug_thingy53`, under :ref:`ug_app_dev`.
```

Reply:
Done.